test/cmd/basicresources.sh

```diff
@@ -275,6 +275,11 @@ os::cmd::expect_success_and_not_text 'oc get is' 'is/test1'
 echo "resource printer: ok"
 os::test::junit::declare_suite_end
 
+os::test::junit::declare_suite_start "cmd/basicresources/login"
```

Comment:
Is `basicresources` where want this? The file is getting bloated. Maybe `authentication`?